src/framework/theme/components/search/search.component.html

```diff
@@ -0,0 +1,18 @@
+    <div class="search-wrap">
+      <button id="btn-search" class="btn btn--search" (click)="searchToggle()">
```

Comment:
Let's remove the id (we don't need it at all, and rename the class to not have '--' in it, just 'btn-search';